pkgs/applications/networking/cluster/cmctl/default.nix

```diff
@@ -1,14 +1,19 @@
 { lib, buildGoModule, fetchFromGitHub, installShellFiles }:
 
-buildGoModule rec {
-  pname = "cmctl";
+let
   version = "1.9.1";
+  rev = "4486c01f726f17d2790a8a563ae6bc6e98465505";
+
+in buildGoModule rec {
+  pname = "cmctl";
+  inherit version;
+  inherit rev;
 
   src = fetchFromGitHub {
+    inherit rev;
     owner = "cert-manager";
     repo = "cert-manager";
-    rev = "v${version}";
-    hash = "sha256-Z1aJ18X4mfJPlCPBC7QgfdX5Tk4+PK8mYoJZhGwz9ec=";
+    sha256 = "1rzm6dn88nc2c8kayg1y9r7gkmbx42s0ph93ji7z56gqqpbqjmk7";
```

Comment:
```suggestion
    rev = "v${version}";
    sha256 = "1rzm6dn88nc2c8kayg1y9r7gkmbx42s0ph93ji7z56gqqpbqjmk7";
```

Reply:
@JoshVanL Can you make this change proposed by Sandro?